packages/services/f-feature-management/src/index.js

```diff
@@ -1,38 +1,107 @@
+import evaluateLogic from './evaluateLogic';
+import evaluateDefault from './evaluateDefaultValue';
+import { getFeatures, init as initFeatures } from './featureGetter';
+import { getContext, setContextGetter } from './contextGetter';
+import trackExperiment from './trackExperiment';
+import { logger, setLogger } from './logger';
+
 const logValueRequest = (func, key, value) => {
-    console.log(`${func.name}: ${key} => ${value}`); // es-lint-disable
+    logger.logInfo(`${func.name}: ${key} => ${value}`);
 };
 
+
+function getValue (key) {
+    logValueRequest(getValue, key, null);
```

Comment:
I'd probably consider abstracting some of the additional functions out of the `index.js` into modules you import in – I'm a bit of a fan of keeping the main entry point as small as possible, just as I think it makes the base module more readable/understandable for anyone new to the code.

Reply:
Done